tox.ini

```diff
@@ -169,7 +169,7 @@ commands=
 sitepackages = false
 skip_install = true
 deps =
-    black==21.6b0
+    black>=21.6b0
```

Comment:
I'm not sure this is useful, better to require the specific version fixing the issue
with click instead of a range.

Reply:
We should always use specific version as any new one might break the formatting out of nowhere.